tfjs-backend-webgpu/src/kernels/conv2d_mm_webgpu.ts

```diff
@@ -31,7 +31,7 @@ export class Conv2DMMProgram implements WebGPUProgram {
   variableNames = ['x', 'W'];
   uniforms = 'ivec2 filterDims, pad, stride;';
   workGroupSize: [number, number, number] = [
-    16, 16,  // must be square (for matmul)
+    16, 16,  // x % y must be zero.
     1
```

Comment:
I'm a little bit confused - in your comment you say that x % y must be 0. But doesn't `computeWorkGroupSize` below make x < y sometimes? 

Can you assert that x % y is zero, and also that x > y? Also, maybe this comment belongs in the matMul shader instead?